### ISSUE TITLE
Add alternative instructions to git clone to instead download zip

### ISSUE THREAD

**Issue by mbroz2** [OpenLiberty/guides-common#290]
Currently, most, if not all, guides instruct the user to 'git clone' the repo.  This mandates that git is installed (a prereq) in order to accomplish the guide.  However, unless the user has to actually make git opperations (I don't think we have a guide like that), the user could just download the zip of the repo instead, which does not need either git install nor github account.  
As such we should add a statement like "Alternatively, if you do not have git installed, download the zip [link] and extract it." to the getting started section of the guides.

This way, we won't have to list "Git" as a prereq to all the guides.

**Comment by gkwan-ibm** [comment]
I agree that it is an alternative to download and unzip the repo, and we can consider to add this to the "Getting started" section. But, I did not see why we can remove  "git" from the prereq list, unless we totally use the unzip way for all the static guides.

**Comment by mbroz2** [comment]
I wasn't clear; I don't mean to remove git from the prereq in the guide ToC/content.  
What I meant is that we won't have to list in the prerequisite section that we're adding to the right hand side in https://github.com/OpenLiberty/openliberty.io/issues/1095

**Comment by gkwan-ibm** [comment]
I see that the Git is not shown on the right top from the screenshot. What if people what to use/know how to get/install Git? 

I am thinking for servicing that we should still encourage readers to use Git instead of unzip. In case there is an issue, we can simply to ask them to run `git pull`.

**Comment by mbroz2** [comment]
We are encouraging them to use git because it's the first step in the content and unzip is the alternative if they don't have it or want it.  The flow described regarding updates over git pull seems unlikely; how many times have we done this in the last 1.5 years?  In order for the user to open an issue against us to tell us something is wrong, they'd need a github account, which means they most likely already have git installed.  

I don't think we should list git in prereqs on the right hand pane, because it's not a prereq if we show the unzip path.  If they don't have git installed, why bother spending the time to get it installed if they don't need it for the guide?  It just adds to noise and falsely raises the barrier to entry for the guides (we should be helping them to focus on the guide and its content as much as possible, and minimizing/simplifying everything else).

We could link to git download/install instructions in the guide content, so we could alter the extra sentence to read something like "If you do not have git installed, you can download it here [link to instructions] or, alternatively, simply download the zip of the project [link] and extract it." This would be my preference and there's precedence in guides to link to other sites from the guide content.

Alternatively, we could try adding an 'Optional' set of prereqs on the right hand side and list git there, but I'd avoid doing that (if we did want to pursue that, we'd need to work with design).

We expect the overwhelming number of guide users to already have git installed, to whom this would not have any impact.  However, for those few that don't (and/or can't), it provides them with another means to do the guide.
